apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html

```diff
@@ -72,9 +77,14 @@ <h5>
                         <h5>
                             {{localeData?.total_tax_amount}}
                             <div>
-                                <span class="d-inline-flex number-total">
+                                <!-- <span class="d-inline-flex number-total">
                                     {{activeCompany?.baseCurrencySymbol}} {{reverseChargeReportResults?.totalTaxAmount |
                                     giddhCurrency}}/-
+                                </span> -->
+                                <span class="d-inline-flex number-total">
+                                    <amount-field [amount]="reverseChargeReportResults?.totalTaxAmount" [currencySymbol]="false"
```

Comment:
Do check currencySymbol is not provided but currencyCode is provided. Check this for Gulf country company

Reply:
added